server/webapp/WEB-INF/rails.new/app/controllers/api_v1/users_controller.rb

```diff
@@ -43,12 +37,48 @@ def update
       end
     end
 
+    def create
+      result = HttpLocalizedOperationResult.new
+
+      user, created = user_service.withEnableUserMutex do
```

Comment:
As much as I'd love for it to be, the method `save` from the controller cannot be moved into java without writing about 100 lines of java code :(